models/bitcask_smiresults_persister.go

```diff
@@ -80,9 +82,10 @@ RETRY:
 
 	start := page * pageSize
 	end := (page+1)*pageSize - 1
-	logrus.Debugf("received page: %d, page size: %d, total: %d", page, pageSize, total)
-	logrus.Debugf("computed start index: %d, end index: %d", start, end)
-
+	//logrus.Debugf("received page: %d, page size: %d, total: %d", page, pageSize, total)
+	s.log.Debug("received page: ", page, " page size: ", pageSize, "total:", total)
```

Comment:
`BitCaskSmiResultsPersister` doesn't have a Logger, this might not work. You can leave `debug` statements as it is for now, let them use `logurus`.